VKappProject/VKappProject/Controllers/FriendCollectionViewController.swift

```diff
@@ -1,23 +1,31 @@
 // FriendCollectionViewController.swift
 // Copyright © RoadMap. All rights reserved.
 
+import Alamofire
 import UIKit
 
 /// Контроллер с фотографией друга
 final class FriendCollectionViewController: UICollectionViewController {
-    // MARK: - Public Property
+    // MARK: - Public Properties
 
-    var friendAvatarName = ""
+    var id = Constants.Items.emptyString
 
-    // MARK: - Private Properies
+    // MARK: - Private Properties
 
-    private let images = [
-        UIImage(named: Constants.UserImageNames.ireneNormanImageName),
-        UIImage(named: Constants.UserImageNames.emilieRivasImageName),
-        UIImage(named: Constants.UserImageNames.connorLloydImageName)
-    ]
+    private var images: [PhotoItem] = [] {
```

Comment:
```suggestion
    private var photoItems: [PhotoItem] = [] {
```
названия должны совпадать